### ISSUE TITLE
owncloud sync - Error 500

### ISSUE THREAD

**Issue by fraunos** [jobisoft/DAV-4-TbSync#28]
Hi, I'm getting error 500 - what could be the case?

**Comment by jobisoft** [comment]
How did you set it up, what Server did you add, can I get a debug log?
https://github.com/jobisoft/TbSync/wiki/Found-a-bug%3F

John

**Comment by jobisoft** [comment]
In general Onwlcloud works, it looks like your server has set up wrong the autodiscover service. Check for redirects /.well-known/caldav and ./well-known/carddav.

For some reasons, some owncloud installations redirect to a URL containing just "dav" instead to "caldav" and "carddav". 

John

**Comment by jobisoft** [comment]
Those redirects ae defined in .htaccess in the root of your owncloud installation:
  
RewriteRule ^\.well-known/carddav /remote.php/carddav/ [R=301,L]
RewriteRule ^\.well-known/caldav /remote.php/caldav/ [R=301,L]

**Comment by jobisoft** [comment]
Why did you create a negative review AND a GitHub issue? I mean, I know that users without GitHub accounts use the review to try to get my Aattention if something is not working (what the review is NOT to be used for, according to the terms) but you DO have a GitHub account. You know how this works. That is frustrating.

**Comment by fraunos** [comment]
I'm sorry, I acted to quickly and didn't notice github repository at the time. Also I thought 3/5 is rather neutral score. If these are editable I'm going to update it.
Yes, my owncloud installation gives me /dav/ url, so I'll dig in it soon.
Thanks for help

**Comment by jobisoft** [comment]
If a review reduces the overall score, it is a negative review. For me at least. But I think you can still change it.

Looking forward to hear from you, if fixing the redirects fixes the problem for you.

**Comment by fraunos** [comment]
Awesome!!!
Commenting out these rewrites (to /dav/) fixed it. Thanks!

**Comment by jobisoft** [comment]
But there should be some redirects to /caldav/ and /carddav/ Right?
http://sabre.io/dav/service-discovery/

**Comment by fraunos** [comment]
It's actually quite weird. I had some rules similar to ones you sent which I commented out and it worked. They looked like this if I remember right. (I may be wrong).
```
RewriteRule ^/remote.php/carddav /remote.php/dav/ [R=301,L]
RewriteRule ^/rempte.php/caldav /remote.php/dav/ [R=301,L]
```
Then I noticed my hosting provider finally supports 10.0.10 version. After updating .htaccess was also reconfigured and has Redirects you mentioned.
Checked it and it just works now.
